spec/models/court_spec.rb

```diff
@@ -70,12 +70,12 @@
     meetup_2 = FactoryGirl.create(
       :meetup,
       court: court,
-      start_time: Time.now + 300000
+      start_time: (Time.now - 4.hours + 3.days)
     )
     meetup_3 = FactoryGirl.create(
       :meetup,
       court: court,
-      start_time: Time.now + 100000
+      start_time: (Time.now - 4.hours + 2.days)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.